src/components/beneficiaries/BeneficiariesFormComponent.tsx

```diff
@@ -12,9 +12,10 @@ function BeneficiariesFormComponent() {
      */
     const getData = () => {
         return {
-            form: VOLUNTEER_FORM,
-            initialValues: VOLUNTEER_INITAL_VALUES,
-            schema: VOLUNTEER_SCHEMA,
+            name: 'Beneficiaries form',
```

Comment:
I don't see `name` being used anywhere - do you want to make this function return:
`return (
    <>
        <h2>{name}</h2>
        <FormikComponent ... />
    </>
);`

Reply:
I set it there without using it as I'm expecting to use maybe in the next updates.